app/assets/javascripts/experiments.js.coffee

```diff
@@ -278,6 +279,25 @@ class window.ExperimentMonitor
     #  ).attr('selected', true)
 
 
+  update_results: (results_info) =>
+    # space (third argument) is restricted to 10 characters
+    results = results_info['results']
+    if results != null
+      results = JSON.stringify(results, null, "&nbsp;")
```

Comment:
W linijkach 286-288 jest kilka wywołań. Wolałbym, żeby to było upakowane w jedną spójną funkcję, która przyjmuje obiekt JSON i zwraca stringa, który jest gotowym do wyświetlenia HTML. Będziemy chcieli popracować nad wyglądem tego, a wtedy wystarczy zmienić implementację tej funkcji i co więcej można to łatwo testować.